source/agent/analytics/videoGstPipeline/VideoGstAnalyzer.cc

```diff
@@ -79,6 +79,18 @@ static void dump(void* index, uint8_t* buf, int len)
     }
 }
 
+static void mem_put_le16(uint8_t *mem, unsigned int val) {
```

Comment:
same functions should not be defined twice. If multiple files are using them, move to a common util .h.